curator-recipes/src/test/java/org/apache/curator/framework/recipes/locks/TestInterProcessReadWriteLock.java

```diff
@@ -362,4 +362,77 @@ private void doLocking(InterProcessLock lock, AtomicInteger concurrentCount, Ato
             }
         }
     }
+
+    public static class LockPathInterProcessReadWriteLock extends InterProcessReadWriteLock
```

Comment:
This class seems not used.

Are you missing a commit?
Is there any test that covers these new APIs?

Reply:
Yes, it is not used, as I have only added it to demonstrate that there is a way to subclass `InterProcessReadWriteLock` the desired way. I will add a test today.